src/renderer/graph/FrequencyResponseChart.tsx

```diff
@@ -21,41 +42,131 @@ const FrequencyResponseChart = () => {
     useAquaContext();
   const prevFilters = useRef<IFilter[]>([]);
   const prevFilterLines = useRef<ChartDataPoint[][]>([]);
+  const prevFreqPoints = useRef<ChartDataPointWithId[][]>([]);
 
   const {
     chartData,
+    pointData,
     autoPreAmpValue,
-  }: { chartData: ChartData[]; autoPreAmpValue: number } = useMemo(() => {
+  }: {
```

Comment:
In general, we need to rename a lot of things here. I'm finding this very hard to read.

Reply:
I took a stab at improving this (mostly by removing unnecessary code and using more consistent terminology). Feedback would be appreciated